curation_concerns-models/app/models/concerns/curation_concerns/generic_file/derivatives.rb

```diff
@@ -0,0 +1,33 @@
+module CurationConcerns
+  module GenericFile
+    module Derivatives
+      extend ActiveSupport::Concern
+
+      included do
+        include Hydra::Derivatives
+        # This was taken directly from Sufia's GenericFile::Derivative.
+        # Using File and ServiceFile so this class will not generate two derivatives 
+        # with the same RDF URI. Is there a mime type ontology we should be using instead?
+        directly_contains_one :ogg, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
Where are these four lines from?  I'm not sure we should be injecting these into every `GenericFile` unless some config value is set (that's the current approach in Sufia) that lets folks express that they care about A/V transcoding. I'm all for testing that this works but I'd prefer to take an opt-in approach rather than creating four possibly extraneous DirectContainers for every GF.